bitnami/node-exporter/README.md

```diff
@@ -0,0 +1,141 @@
+| `rbac.create`                        | Wether to create & use RBAC resources or not                                                             | `true`                                                                    |
+| `rbac.apiVersion`                    | Version of the RBAC API                                                                                  | `v1beta1`                                                                 |
+| `rbac.pspEnabled`                    | PodSecurityPolicy                                                                                        | `true`                                                                    |
+| `image.registry`                     | Node Exporter image registry                                                                             | `docker.io`                                                               |
+| `image.repository`                   | Node Exporter Image name                                                                                 | `bitnami/node-exporter`                                                   |
+| `image.tag`                          | Node Exporter Image tag                                                                                  | `{TAG_NAME}`                                                              |
+| `image.pullPolicy`                   | Node Exporter image pull policy                                                                          | `IfNotPresent`                                                            |
+| `image.pullSecrets`                  | Specify docker-registry secret names as an array                                                         | `[]` (does not add image pull secrets to deployed pods)                   |
+| `extraArgs`                          | Additional command line arguments to pass to node-exporter                                               | `{}`                                                                      |
+| `extraVolumes`                       | Additional volumes to the node-exporter pods                                                             | `{}`                                                                      |
+| `extraVolumeMounts`                  | Additional volumeMounts to the node-exporter container                                                   | `{}`                                                                      |
+| `serviceAccount.create`              | Specify whether to create a ServiceAccount for Node Exporter                                             | `true`                                                                    |
+| `serviceAccount.name`                | The name of the ServiceAccount to create                                                                 | Generated using the `node-exporter.fullname` template                     |
```

Comment:
I would move the `serviceAccount` parameters just after the `rbac` parameters (they're closely related)